Dockerfile

```diff
@@ -164,6 +162,8 @@ RUN --mount=type=bind,from=build-image,source=/tmp/wheelhouse,target=/mnt/wheelh
 ENV NUMPROCS=1
 COPY --from=build-image-av /opt/ffmpeg/lib /usr/lib
 
+RUN chown -R ${USER} /var/lib/nginx /var/log/nginx
```

Comment:
`/var/log/nginx` shouldn't need to be modified, since you overrode the log file locations to be in `/home/django`. I would also suggest to override the location of whatever it is that is stored to `/var/lib/nginx`, to be stored in `/home/django` too, for consistency.

Reply:
fixed